src/translations/en.json

```
@@ -4181,7 +4181,7 @@
       "open_release_notes": "Open release notes",
       "create_backup": "Create backup before updating",
       "description": "There is an update available for the {name}. You have {version} installed. Click update to update to version {newest_version}",
-      "core_note": "The supervisor will roll back to version {version} if your instance does not come up after the update.",
+      "core_note": "The supervisor will roll back if your instance does not come up after the update.",
```

Let's drop "Supervisor".

```suggestion
      "core_note": "It's safe to update. Your system will automatically restore the previous version if the new version fails to start.",
```

hmm thinking about this now, maybe we should just drop it because why would we even raise the idea that it could go wrong?

---

I vote to keep it, since it gives more confidence to the user to update, knowing that the system is capable of correcting itself.

---

Ludeeus and I talked offline and decided we should drop it because a user is going to wonder "Hey, why is that warning there to begin with"